decidim-admin/app/controllers/decidim/admin/features_controller.rb

```diff
@@ -20,7 +20,13 @@ def index
       def new
         authorize! :create, Feature
 
-        @form = form(FeatureForm).instance(name: default_name(manifest))
+        @feature = Feature.new(
```

Comment:
I'm really struggling here trying to work out the way to serialize data into the form in an elegant way. This is needed since we want to prepopulate the name and the form expects it coming as a hash from the model. Otherwise, since the whole `TranslatableAttributes` is a hack, would have to tamper with the attributes.

I'll have to refactor all this at some point, but right now, I'd feel much more comfortable shipping this so other modules can depend on it.